application/backend/src/main/resources/apps/typerefinery/components/graphs/barchart/clientlibs/functions.js

```diff
@@ -32,8 +32,14 @@ const DEFAULT_BAR_CHART_DATA = {
     "PsExec",
     "Minikatz",
   ],
-  dataSetBorderColor:   Typerefinery?.Theme?.rootElementStyle?.getPropertyValue('--border-color') || "#0099DE",
-  canvasBackgroundColor: Typerefinery?.Theme?.rootElementStyle?.getPropertyValue('--primary-object-background-color') || "#343a40",
+  dataSetBorderColor:
+    Typerefinery?.Theme?.rootElementStyle?.getPropertyValue(
+      "--border-color"
+    ) || "#0099DE",
```

Comment:
can we please convert all of these # values into CSS modifiers so that we don't hardcode CSS into code as its to hard to update, clearly now if we want to change colours we have to update JS!